core/src/model.cpp

```diff
@@ -18,4 +18,19 @@ std::unique_ptr<Vector> Model::apply(Vector& in) {
 
 }
 
+void Model::add_testing_data(std::vector<Vector> ins, std::vector<Vector> outs) {
+    inputs .insert(std::end(inputs),  std::begin(ins),  std::end(ins));
```

Comment:
I didn't know you could have a space between these

Reply:
I just did it so it will visually line up with the line below. Do you think it’s bad?

Reply:
It's totally fine, just haven't seen it before. It does look nice